src/prognostic_equations/edmfx_sgs_flux.jl

```diff
@@ -118,7 +176,19 @@ function edmfx_sgs_flux_tendency!(
             )
         end
 
+        # diffusive flux
+        ᶠρaK_h = p.ᶠtemp_scalar
+        @. ᶠρaK_h[colidx] = ᶠinterp(Y.c.ρ[colidx]) * ᶠinterp(ᶜK_h[colidx])
+
+        ᶜdivᵥ_ρe_tot = Operators.DivergenceF2C(
+            top = Operators.SetValue(C3(FT(0))),
+            bottom = Operators.SetValue(sfc_conditions.ρ_flux_h_tot[colidx]),
+        )
+        @. Yₜ.c.ρe_tot[colidx] -=
+            ᶜdivᵥ_ρe_tot(-(ᶠρaK_h[colidx] * ᶠgradᵥ(ᶜh_tot[colidx])))
+
```

Comment:
This has the same form as the equation for the `EDMFX` type. LGTM overall -  `ᶜh_tot[colidx]` here is the value computed from cached dycore variables, and the corresponding enthalpy above in the equation for `EDMFX` 
```
                    TD.total_specific_enthalpy(
                        thermo_params,
                        ᶜts⁰[colidx],
                        ᶜspecific⁰.e_tot[colidx],
                    ),
```
accounts for the subgrid updraft variables (?) - is my understanding right here?

Reply:
It's the environment. I will probably add h_tot^0 to cache in a separate PR.